ui/app/styles/charts/distribution-bar.scss

```diff
@@ -98,8 +97,6 @@
         &.is-clickable {
           a {
             display: block;
-            text-decoration: none;
-            color: inherit;
           }
 
           &:not(.is-empty) {
```

Comment:
Maybe we can remove this rule now? The background colour change on hover may not be necessary anymore.

![image](https://user-images.githubusercontent.com/775380/149255311-82786c7b-5bd2-456a-823a-c7633d9d418d.png)